tests/TableTestCase.php

```diff
@@ -153,9 +153,9 @@ public function testGetData()
 
     public function testSetSequenceName()
     {
-        $this->objTable->sequenceName = 'test-seq';
+        $this->objTable->setOption('sequenceName', 'test-seq');
```

Comment:
This test might have been wrong for a long time. I created the following example to demonstrate, that without using setOption, this test accidentally created a new public property instead of setting the value in the options array:

```php
<?php

class X {
    private $storage = [];
    
    public function __construct() {
        $this->storage['foo'] = 'bar';
    }
    
    public function __get($name) {
        var_dump('#### magic getter!');
        return $this->storage[$name];
    }
    
    public function getStorage($name) {
        return $this->storage[$name];
    }
}


$x = new X();
var_dump($x->foo);                  // Returns the value of X::$storage['foo']
var_dump($x->getStorage('foo'));    // Returns the value of X::$storage['foo']
$x->foo = "baz";                    // Creates a new dynamic property $foo
var_dump($x->foo);                  // Returns the value of the dynamic property $foo
var_dump($x->getStorage('foo'));    // Returns the value of X::$storage['foo']
```